Resource/doc/RequestListener.md

```diff
@@ -0,0 +1,27 @@
+RequestListener
+==================
```

Comment:
why you're doing this? result will be the same if you use `# RequestListener`

Reply:
just based on https://raw.githubusercontent.com/symfony/symfony-standard/3.3/README.md template

Reply:
I was just wondering :)